jenkins/e2e-image/e2e-runner.sh

```diff
@@ -84,6 +84,54 @@ function running_in_docker() {
     grep -q docker /proc/self/cgroup
 }
 
+### FUNCTIONS TO SUPPORT get-kube.sh ###
+
+# Moves local tarballs into a local path suitable for get-kube.sh, and sets
+# KUBERNETES_RELEASE and KUBERNETES_RELEASE_URL accordingly.
+function set_release_vars_from_local_gcs_stage() {
+    local -r local_gcs_stage_path="${WORKSPACE}/_output/gcs-stage"
+    KUBERNETES_RELEASE_URL="file://${local_gcs_stage_path}"
+    KUBERNETES_RELEASE=$(ls "${local_gcs_stage_path}" | grep ^v.*$)
+    if [[ -z "${KUBERNETES_RELEASE}" ]]; then
+      echo "FAIL! version not found in ${local_gcs_stage_path}"
+      return 1
+    fi
+}
+
+# Use a published version like "ci/latest" (default), "release/latest",
+# "release/latest-1", or "release/stable".
+# TODO(ixdy): maybe this should be in get-kube.sh?
+function set_release_vars_from_gcs() {
```

Comment:
it may not be clear, but this function and the next one (for GKE) are just slightly modified versions of existing functions.